frontend/public/components/modals/delete-modal.tsx

```diff
@@ -24,10 +19,11 @@ import { k8sList } from '../../module/k8s/resource';
 import { ResourceLink } from '../utils/resource-link';
 
 //Modal for resource deletion and allows cascading deletes if propagationPolicy is provided for the enum
-const DeleteModal = withHandlePromise((props: DeleteModalProps) => {
```

Comment:
Is `withHandlePromise` not working? We use this in a lot of places, so I'm worried it's a problem elsewhere.

Reply:
It seems to handle the promise okay, but I couldn't get error handling to work with it.

Reply:
withHandlePromise looked ok. I think I figured out the real issue.